scripts/postinstall.sh

```diff
@@ -3,3 +3,6 @@
 set -e
 
 cp node_modules/testpilot-metrics/testpilot-metrics.js src/vendor
+cp node_modules/kinto-http/dist/kinto-http.js src/sidebar/vendor
+cp node_modules/quill/dist/quill.js src/sidebar/vendor
```

Comment:
Not sure about your feelings on using _quill.js (409K)_ vs _quill.**min**.js (202K)_. 🤷‍♀️ 

```sh
➜  notes git:(master) ✗ ls -lash node_modules/quill/dist

  56 -rw-r--r--   1 pdehaan  staff    24K Jun  5 10:21 quill.bubble.css
  24 -rw-r--r--   1 pdehaan  staff   8.6K Jun  5 10:21 quill.core.css
 568 -rw-r--r--   1 pdehaan  staff   283K Jun  5 10:21 quill.core.js
 824 -rw-r--r--   1 pdehaan  staff   409K Jun  5 10:21 quill.js
 408 -rw-r--r--   1 pdehaan  staff   202K Jun  5 10:21 quill.min.js
1904 -rw-r--r--   1 pdehaan  staff   951K Jun  5 10:21 quill.min.js.map
  48 -rw-r--r--   1 pdehaan  staff    24K Jun  5 10:21 quill.snow.css
```